src/Pagination/Pagination.tsx

```diff
@@ -3,18 +3,26 @@ import type { ComponentProps } from 'react';
 import { cn } from '@/lib/utils';
 import type { TableView } from '@/types';
 
+import { SelectInput } from '@/BasicInputs';
 import * as Shadcn from '../lib/components/ui/pagination';
-
 export interface PaginationProps
   extends ComponentProps<typeof Shadcn.Pagination>,
     Pick<TableView, 'page'> {
   totalDataLength?: number;
-  perPage?: TableView['per_page'];
+  perPage?: TableView['perPage'];
   maxPageButtonLimit?: number; // The max number of page btns to show at a time
   onPageChange: (newPage: TableView['page']) => void;
-  onPerPageChange?: (newPerPage: TableView['per_page']) => void;
+  onPerPageChange?: (newPerPage: TableView['perPage']) => void;
 }
 
+const PER_PAGE_VALUES = [
+  { value: 5, label: '5' },
+  { value: 10, label: '10' },
+  { value: 25, label: '25' },
+  { value: 50, label: '50' },
+  { value: 100, label: '100' },
+];
+
```

Comment:
Make it so that a dev can specify the options, maybe a prop for `perPageOptions: SelectOption[]`

Reply:
Okay I created a new interface for the prop because I wanted to make sure that it is getting a number in the value attribute specifically.